storage.bs

```diff
@@ -151,16 +151,30 @@ must be cleared in its entirety.
 <h2 id=persistence>Persistence permission</h2>
 
 A <a>box</a> can only be turned into a <a>persistent box</a> if the user (or user agent
-on behalf of the user) has granted <dfn>permission</dfn>.
+on behalf of the user) has granted permission to use the {{"persistent-storage"}} feature.
 
 <p class="note">When granted to an <a>origin</a>, the persistence permission can be used to protect
 storage from the user agent's clearing policies. The user agent cannot clear storage marked as
 persistent without involvement from the <a>origin</a> or user. This makes it particularly useful for
 resources the user needs to have available while offline or resources the user creates locally.
 
-<p class="XXX">We will eventually integrate with the Permissions API, with the identifier
-"<dfn export for=PermissionName type=enum-value><code>persistent-storage</code></dfn>", but since
-that specification is not in great shape at the moment that has not happened yet.
+The <dfn for="PermissionName" enum-value>"persistent-storage"</dfn> <a>powerful feature</a>'s
```

Comment:
I reintroduced it in https://github.com/w3c/permissions/pull/97, but with a more tautological definition: ["A feature of a UA that some code might not be allowed to access, for example because its environment settings object doesn’t satisfy some criteria, or because the user hasn’t given permission."](https://w3c.github.io/permissions/#powerful-feature)

@mikewest was happy with it, and because it doesn't include the "should only be accessible from HTTPS" bits, I don't expect other folks to have a problem. If they do, we could back off to just "feature". And we might do that anyway if we unify the PermissionName enum with the [list in Feature Policy](https://wicg.github.io/feature-policy/#features).